src/sql/workbench/contrib/queryplan2/browser/graphElementPropertiesView.ts

```diff
@@ -0,0 +1,241 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as DOM from 'vs/base/browser/dom';
+import * as azdata from 'azdata';
+import { localize } from 'vs/nls';
+import { Action } from 'vs/base/common/actions';
+import { Codicon } from 'vs/base/common/codicons';
+import { IThemeService } from 'vs/platform/theme/common/themeService';
+import { attachTableStyler } from 'sql/platform/theme/common/styler';
+import { TableDataView } from 'sql/base/browser/ui/table/tableDataView';
+import { Table } from 'sql/base/browser/ui/table/table';
+import { RESULTS_GRID_DEFAULTS } from 'sql/workbench/common/constants';
+import { isString } from 'vs/base/common/types';
+import { ActionBar } from 'sql/base/browser/ui/taskbar/actionbar';
+import { ActionsOrientation } from 'vs/base/browser/ui/actionbar/actionbar';
+
+
+export class GraphElementPropertiesView {
+
+	// Title bar with close button action
+	private _propertiesTitle!: HTMLElement;
+	private _titleText!: HTMLElement;
+	private _titleActionBarContainer!: HTMLElement;
+	private _titleActionBar: ActionBar;
+
+	// Div that holds the name of the element selected
+	private _operationName!: HTMLElement;
+
+	// Action bar that contains sorting option for the table
+	private _tableActionBarContainer!: HTMLElement;
+	private _tableActionBar!: ActionBar;
+
+	// Properties table
+	private _table: Table<any>;
+	private _dataView: TableDataView<Slick.SlickData>;
+	private _data: { [key: string]: string }[];
+	private _tableContainer!: HTMLElement;
+	private _actualTable!: HTMLElement;
+
+	// Table dimensions.
+	private _tableWidth = 485;
```

Comment:
how are these values being determined?

Reply:
I have used properties view width as a reference point while leaving some space for the table scroll bar.
For the height, it is the leftover space in the properties view.